src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -703,6 +704,27 @@ public function iChangeMyPasswordTo($newPassword)
         $this->pressButton('Save changes');
     }
 
+    /**
+     * @When I remove :name from cart
+     */
+    public function iRemoveFromCart($name)
+    {
+        $name = explode(' ', $name)[0];
+        $trLocator = sprintf('table tbody tr:contains("%s")', $name);
+        $tr = $this->getElementByCss($trLocator);
+        $buttonLocator = sprintf('td a.btn-danger');
+        $button = $tr->find('css', $buttonLocator);
+        $button->click();
```

Comment:
Yes it is quite weird locator. I thought about adding something in view but i wasn't sure. Thx